apps/editors/models.py

```diff
@@ -395,6 +396,21 @@ def award_points(cls, user, addon, status, **kwargs):
         """
         event = cls.get_event(addon, status, **kwargs)
         score = amo.REVIEWED_SCORES.get(event)
+
+        try:
+            vq = ViewQueue.objects.get(
+                addon_name=addon.name,
```

Comment:
I don't think you need the addon name, which isn't unique. The slug is however, and should be sufficient.